shared/src/test/java/TestStatsProcessor.java

```diff
@@ -57,4 +57,18 @@ public void getLeaderboard() {
         assertThat(leaderboard[1].getPlayerName(), equalTo("CHS"));
         assertThat(leaderboard[1].getHighestScore(), equalTo(18));
     }
+
+    @Test
+    public void getEmptyAllStats() {
+        Database db = Database.create("concrete", "dummyFilename");
```

Comment:
Oh just realized those are different files. You can follow the example of `TestDatabase` in `TestStatsProcessor` to create a temporary file for the database